pom.xml

```diff
@@ -83,6 +83,11 @@
         <version>1.32</version>
       </dependency>
       <dependency>
+        <groupId>com.google.code.gson</groupId>
+        <artifactId>gson</artifactId>
+        <version>2.2.4</version>
```

Comment:
Are you using `2.2.4` due to Hadoop using that version?  Could add comment if Hadoop restricts use of latest version.